UI/GameInfoCache.cpp

```diff
@@ -628,12 +625,14 @@ class GameInfoWorkItem : public PrioritizedWorkQueueItem {
 			info_->installDataSize = info_->GetInstallDataSizeInBytes();
 		}
 
+done:
 		info_->pending = false;
 		info_->working = false;
+		info_->readyEvent.Notify();
```

Comment:
Feels like this should rather be a closure or a private method.  Though I don't treat gotos like a plague, they are harder to read especially for less seasoned programmers... usually they are pretty avoidable.

It's kinda like my opinion on wrapping an if and how to indent the conditions in the additional lines: you've already lost the battle of readability if you're wrapping an if, thinking about indentation is just asking the wrong question.  There's almost always a better way.

-[Unknown]

Reply:
Agreed, I'll refactor this away before we get even close to merging this.